python-checks/src/main/java/org/sonar/python/checks/AllBranchesAreIdenticalCheck.java

```diff
@@ -72,17 +74,17 @@ private static void handleIfStatement(IfStatement ifStmt, SubscriptionContext ct
       return;
     }
     IssueWithQuickFix issue = (IssueWithQuickFix) ctx.addIssue(ifStmt.keyword(), IF_STATEMENT_MESSAGE);
-    issue.secondary(issueLocation(ifStmt.body()));
-    ifStmt.elifBranches().forEach(e -> issue.secondary(issueLocation(e.body())));
-    issue.secondary(issueLocation(elseBranch.body()));
+    issue.secondary(issueLocation(ifStmt.body(), SECONDARY_MESSAGE));
+    ifStmt.elifBranches().forEach(e -> issue.secondary(issueLocation(e.body(), SECONDARY_MESSAGE)));
+    issue.secondary(issueLocation(elseBranch.body(), SECONDARY_MESSAGE));
     if (!hasSideEffect(ifStmt)) {
       issue.addQuickFix(computeQuickFixForIfStatement(ifStmt, elseBranch));
     }
   }
 
-  private static IssueLocation issueLocation(StatementList body) {
+  private static IssueLocation issueLocation(StatementList body, @Nullable String message) {
```

Comment:
It seems that this method only deals with secondary locations. We could consider renaming it `secondaryLocation` instead (as you prefer). Also, you can probably remove the `message` parameter and inline `SECONDARY_MESSAGE`, since the value of the parameter is always the same.

Reply:
done